pkgs/tools/networking/openconnect/common.nix

```diff
@@ -0,0 +1,45 @@
+{ version
+, src
+, useAutoreconfHook ? false
+}:
+
+{ lib
+, stdenv
+, pkg-config
+, gnutls
+, openssl
+, useOpenSSL ? false
+, gmp
+, libxml2
+, stoken
+, zlib
+, vpnc-scripts
+, darwin
+, autoreconfHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "openconnect";
+  inherit version src;
+
+  outputs = [ "out" "dev" ];
+
+  configureFlags = [
+    "--with-vpnc-script=${vpnc-scripts}/bin/vpnc-script"
+    "--disable-nls"
+    "--without-openssl-version-check"
+  ];
+
+  buildInputs = [ gmp libxml2 stoken zlib (if useOpenSSL then openssl else gnutls) ]
+    ++ lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.PCSC;
```

Comment:
```suggestion
    ++ lib.optional stdenv.isDarwin PCSC;
```

Reply:
If I'm not mistaken, this is needed because the attrset passed to the function now is `darwin`, and not `PCSC`

Reply:
PCSC should be in the inputs and inherited from darwin.